mods/common/languages/en.ftl

```diff
@@ -172,5 +283,289 @@ none = None
+delete-all-maps-prompt = Delete all maps on this page?
+delete-all-maps-accept = Delete
+
+## MissionBrowserLogic
+no-video-title = Video not installed
+no-video-text = The game videos can be installed from the
+    "Manage Content" menu in the mod chooser.
+no-video-cancel = Back
+
+cant-play-title = Unable to play video
+cant-play-prompt = Something went wrong during video playback.
+cant-play-cancel = Back
+
+## MusicPlayerLogic
+sound-muted = Audio has been muted in settings.
+no-song-playing = No song playing
+
+## MuteHotkeyLogic
+audio-muted = Audio muted.
+audio-unmuted = Audio unmuted.
+
+## PlayerProfileLogic
+loading-player-profile = Loading player profile...
+loading-player-profile-failed = Failed to load player profile.
+
+## ReplayBrowserLogic
+duration = Duration: { $time }
+singleplayer = Singleplayer
+multiplayer = Multiplayer
+
+today = Today
+last-week = Last 7 days
+last-fortnight = Last 14 days
+last-month = Last 30 days
+
+replay-duration-very-short = Under 5 min
+replay-duration-short = Short (10 min)
+replay-duration-medium = Medium (30 min)
+replay-duration-long = Long (60+ min)
+
+rename-replay-title = Rename Replay
+rename-replay-prompt = Enter a new file name:
+rename-replay-accept = Rename
+
+delete-replay-title = Delete selected replay?
+delete-replay-prompt = Delete replay { $replay }?
+delete-replay-accept = Delete
+
+delete-all-replays-title = Delete all selected replays?
+delete-all-replays-prompt =
+    { $count ->
+        [one] Delete { $count } replay.
+       *[other] Delete { $count } replays.
+    }
+delete-all-replays-accept = Delete All
+
+replay-deletion-failed = Failed to delete replay file '{ $file }'. See the debug.log file for details.
+
+## ReplayUtils
+incompatible-replay-title = Incompatible Replay
+incompatible-replay-prompt = Replay metadata could not be read.
+-incompatible-replay-recorded = It was recorded with an
+incompatible-replay-unknown-version = { -incompatible-replay-recorded } unknown version: { $version }.
+incompatible-replay-unknown-mod = { -incompatible-replay-recorded } unknown mod: { $mod }.
+incompatible-replay-unavailable-mod = { -incompatible-replay-recorded } unavailable mod: { $mod }.
+incompatible-replay-incompatible-version = { -incompatible-replay-recorded } incompatible version: { $version }.
+incompatible-replay-unavailable-map = { -incompatible-replay-recorded } unavailable map: { $map }.
+
+## ServerCreationLogic
+internet-server-nat-A = Internet Server (UPnP/NAT-PMP
+internet-server-nat-B-enabled = Enabled
+internet-server-nat-B-not-supported = Not Supported
+internet-server-nat-B-disabled = Disabled
+internet-server-nat-C = ):
+
+local-server = Local Server:
+
+server-creation-failed-prompt = Could not listen on port { $port }
+server-creation-failed-port-used = Check if the port is already being used.
+server-creation-failed-error = Error is: "{ $message }" ({ $code })
+server-creation-failed-title = Server Creation Failed
+server-creation-failed-cancel = Back
```

Comment:
"Back" is a great example of a common UI string that are better to be consolidated to a single entry in the translation file. Not necessary a blocker for this PR but something to consider seriously.

Reply:
Addressed in #20207